src/LinearSolvers/blockIterativeWrapper.jl

```diff
@@ -92,11 +92,11 @@ function solveLinearSystem!(A,B,X,param::BlockIterativeSolver,doTranspose=0)
 		if param.PC==:ssor
 			OmInvD = 1./diag(A);
 			Xt      = zeros(n,nrhs)
-			M(R)   = (Xt[:]=0.0; tic(); Xt=ssorPrecTrans!(A,Xt,R,OmInvD); param.timePC+=toq(); return Xt);
+			M = R -> (Xt[:]=0.0; tic(); Xt=ssorPrecTrans!(A,Xt,R,OmInvD); param.timePC+=toq(); return Xt);
```

Comment:
Yes, in this context there is now an important difference. The problem is that conditionally adding methods to functions doesn't work in 0.5. However, you can get effectively the same behaviour using anonymous functions. See the discussion in the following issue: https://github.com/JuliaLang/julia/issues/15602